hooks/src/index.js

```diff
@@ -222,6 +222,25 @@ export function useDebugValue(value, formatter) {
 	}
 }
 
+export function errorBoundary(fn, callback) {
+	let setErr;
```

Comment:
I think this breaks if the component is rendered in two different places, since they would both assign to setErr. Maybe a decent argument for not having this produce its own state? (callback-only)

Reply:
Problem being that we can't set the state in the callback (second argument) so we can't render a fallback which could result in endless loops